feature/client/src/commonMain/kotlin/com/mifos/feature/client/clientLoanAccounts/ClientLoanAccountsRoute.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025 Mifos Initiative
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * See https://github.com/openMF/android-client/blob/master/LICENSE.md
+ */
+package com.mifos.feature.client.clientLoanAccounts
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ClientLoanAccountsRoute(
+    val clientId: Int = -1,
+)
+
+fun NavGraphBuilder.clientLoanAccountsDestination(
+    navigateBack: () -> Unit,
+    navigateToViewAccount: (Int) -> Unit,
+) {
+    composable<ClientLoanAccountsRoute> {
+        ClientLoanAccountsScreenRoute(
```

Comment:
Can be changed as follows as it is a composable func that refers to a screen not a route
```suggestion
        ClientLoanAccountsScreen(
```

Reply:
It was added to distinguish between the two `ClientLoanAccountsScreen`